lib/rspec/core/example.rb

```diff
@@ -225,10 +225,7 @@ def run(example_group_instance, reporter)
         rescue Exception => e
           set_exception(e)
         ensure
-          ExampleGroup.each_instance_variable_for_example(@example_group_instance) do |ivar|
-            @example_group_instance.instance_variable_set(ivar, nil)
-          end
-          @example_group_instance = nil
+          @example_group_instance = nil # if you love something... let it go
```

Comment:
But it makes @JoshCheek super happy!

https://twitter.com/josh_cheek/status/592937805636050944